onnxscript/rewriter/generic_pattern.py

```diff
@@ -477,10 +549,19 @@ def match_pattern(
             f"Class {cls.__name__!r} must overwrite method match_pattern."
         )
 
-    @classmethod
     def _build_pattern(
-        cls, g: ModelWithGraphStructure, fct: typing.Callable
+        self,
+        g: ModelWithGraphStructure,
+        fct: typing.Callable | None = None,
```

Comment:
Good time to rename to avoid abbreviations

Reply:
I'd rather create another PR to do that since this PR was started: https://github.com/microsoft/onnxscript/pull/1369.